src/components/Dropdown/Dropdown.tsx

```diff
@@ -118,51 +118,66 @@ const ItemFilter = ({
                 }}
                 active={isOpen}
             >
-                <Button
-                    className={`item-filter-button ${isOpen ? " open" : ""}`}
-                    buttonType="outline"
-                    onClick={clickHandler}
-                    type="button"
-                >
-                    {filter}
-                    {numOfSelections ? ` (${numOfSelections})` : null}{" "}
-                    <Icon name={isOpen ? "minus" : "plus"} />
-                </Button>
-                {open ? (
-                    <div className="item-filter-content">
-                        <fieldset>
-                            {distinctOptions.map((option, i) => (
-                                <Checkbox
-                                    labelOptions={{
-                                        id: option.id,
-                                        labelContent: option.label,
-                                    }}
-                                    onChange={() => handleCheckbox(option)}
-                                    key={option.id || i}
-                                    isSelected={isSelected(option)}
-                                />
-                            ))}
-                        </fieldset>
-                        {!mobile ? (
-                            <div className="item-filter-buttons">
-                                <Button
-                                    buttonType="link"
-                                    onClick={() => manageFilterDisplay("none")}
-                                >
-                                    Clear
-                                </Button>
-                                <Button
-                                    onClick={() =>
-                                        submitFilterSelections(selectedFilters)
-                                    }
-                                    disabled={!selectionMade}
-                                >
-                                    Apply
-                                </Button>
-                            </div>
-                        ) : null}
-                    </div>
-                ) : null}
+                <>
```

Comment:
There are no changes here, just that `focus-trap-react` asks for a single child, so I stuck everything under a fragment.